### ISSUE TITLE
Can't get log to work

### ISSUE THREAD

**Issue by chocolateboy** [ingydotnet/git-subrepo#61]
git-subrepo: 0.2.0 (a3f2e53)
git: 1.9.1
OS: Linux (Ubuntu 14.04.1)

I can't get either of the two different commands in the README to give me a log of the subrepo's history:

### setup

```bash
#!/bin/bash -x

dir=subrepo_test

mkdir $dir && cd $dir

echo 'foo' > foo.txt
echo 'bar' > bar.txt

git init .
git stage foo.txt bar.txt
git commit -m 'initial commit'

git subrepo clone https://github.com/alex/what-happens-when.git test/what-happens-when

git log --oneline | cat

# 0ef5bea git subrepo clone https://github.com/alex/what-happens-when.git test/what-happens-when
# d7122ba initial commit
```

### halp!

```bash
git log subrepo/test/what-happens-when/upstream

# fatal: ambiguous argument 'subrepo/test/what-happens-when/upstream': unknown revision or path not in the working tree.
# Use '--' to separate paths from revisions, like this:
# 'git <command> [<revision>...] -- [<file>...]'

git log refs/subrepo/test/what-happens-when/upstream

# fatal: ambiguous argument 'refs/subrepo/test/what-happens-when/upstream': unknown revision or path not in the working tree.
# Use '--' to separate paths from revisions, like this:
# 'git <command> [<revision>...] -- [<file>...]'
```

**Comment by chocolateboy** [comment]
Same result if the repo is cloned from the ssh URL e.g.:

    git subrepo clone git@github.com:alex/what-happens-when.git test/what-happens-when

\- although, in this case, at least `.git/refs/subrepo/test/what-happens-when/` exists.

**Comment by ingydotnet** [comment]
Fixed by 96647af

The ref was supposed to be `.../fetch` not `.../upstream`

Sorry it took a year.
